src/main/java/domain/Position.java

```diff
@@ -0,0 +1,68 @@
+package domain;
+
+import static domain.ExceptionType.POSITION_OVERFLOW;
+
+import java.util.List;
+import java.util.Objects;
+
+class Position {
+    private final int rawPosition;
+    private final int maxPosition;
+
+
+    Position(int rawPosition, int maxPosition) {
+        validateRange(rawPosition, maxPosition);
+        this.rawPosition = rawPosition;
+        this.maxPosition = maxPosition;
+    }
+
+    private void validateRange(int rawPosition, int maxPosition) {
+        if (rawPosition < 0 || rawPosition > maxPosition) {
+            throw new LadderGameException(POSITION_OVERFLOW);
+        }
+    }
+
+    Position move(List<Boolean> bridges) {
+        Boolean canGoRight = getCanGo(bridges, rawPosition);
+        Boolean canGoLeft = getCanGo(bridges, rawPosition - 1);
+        if (canGoRight) {
+            return new Position(rawPosition + 1, maxPosition);
```

Comment:
이 부분도 메서드로 표현해보는건 어떻게 생각하시나요?

```suggestion 
return this.moveRight();
```

```java
private Position moveRight() {
    // ...
}
```